v4m-chart/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: v4m
+description: Viya 4 Monitoring for Kubernetes (https://github.com/sassoftware/viya4-monitoring-kubernetes)
+type: application
+version: 1.0.10-SNAPSHOT
+appVersion: "1.0.10-SNAPSHOT"
+# foo
```

Comment:
What's with the # foo line?

Reply:
oops. was testing the new 'git status' functionality to detect changes to the local repo